conf/log.conf.d/httpd.aaa.conf.example

```diff
@@ -1,21 +1,6 @@
 ### httpd.aaa logger ###
-log4perl.rootLogger = INFO, HTTPD_AAA
```

Comment:
Good point.
This uses info as default.
I will add the config lines to change it.